docs/UserGuide.md

```diff
@@ -176,40 +182,57 @@ Format: `list`
 
 ### Adding a patient: `add`
 
-Adds a patient
+Adds a patient into the system
 
 Format: `add n/NAME i/IC_NUMBER [g/GENDER] [b/BIRTHDAY] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [pr/PRIORITY]] [t/TAG]...`
 
-+ Only `NAME` and `IC_NUMBER` fields are **compulsory**.
-+ **All** input fields are **case-insensitive** e.g. `T1234567A` is the same as `t1234567a`.
-+ The order of the input fields does not matter.
-+ `PRIORITY` can take on values `NIL`,`LOW`,`MEDIUM` or `HIGH`.
++ IC number cannot be edited aftet the patient has been added.
 
-Examples:
+**Caution:** Duplicate patients are not allowed. Duplicate patients refer to patients with the same name and/or IC
+number.
+
+- Names are case-sensitive
+    - Han Bo is different from han bo
```

Comment:
Perhaps "Han Bo" and "han bo" should be `Han Bo` and `han bo` instead, such that it is more in line with our Google Doc UG? same for IC number below